inc/admin.php

```diff
@@ -1356,46 +1356,73 @@ public function get_layout_directories(){
 	public static function display_footer_premium_link() {
 		$links = array(
 			array(
-				'text' => __( 'Get an Accordion Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/accordion' )
-			),			
+				'text' => __( 'Get the row, cell, and widget %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/animations' ),
+				'anchor' => __( 'Animations Addon', 'siteorigin-panels' ),
+			),
+			array(
+				'text' => __( 'Get the %link%. Build custom post types with reusable Page Builder layouts.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/cpt-builder' ),
+				'anchor' => __( 'CPT Builder Addon', 'siteorigin-panels' ),
+			),
+			array(
+				'text' => __( 'Get the %link%. Add beautiful and customizable text overlays with animations to your images.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/image-overlay' ),
+				'anchor' => __( 'Image Overlay Addon', 'siteorigin-panels' ),
+			),
 			array(
-				'text' => __( 'Get the row, cell, and widget Animations Addon', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/animations' )
+				'text' => __( 'Get a %link% for SiteOrigin Image, Masonry, and Slider Widgets.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/lightbox' ),
+				'anchor' => __( 'Lightbox Addon', 'siteorigin-panels' ),
 			),
 			array(
-				'text' => __( 'Get a Carousel Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/carousel' )
+				'text' => __( 'Link an entire Page Builder row, cell, or widget with the %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/link-overlay' ),
+				'anchor' => __( 'Link Overlay Addon', 'siteorigin-panels' ),
 			),
 			array(
-				'text' => __( 'Get a Call to Action Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/cta' )
-			),						
+				'text' => __( 'Get the %link%. Create a widget once, use it everywhere. Update it and the changes reflect in all instances of the widget.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/mirror-widgets' ),
+				'anchor' => __( 'Mirror Widgets Addon', 'siteorigin-panels' ),
+			),
 			array(
-				'text' => __( 'Get a Lightbox Addon for SiteOrigin Widgets', 'siteorigin-panels' ),
-				'url' => SiteOrigin_Panels::premium_url( 'plugin/lightbox' )
+				'text' => __( 'Upload multiple image frames at once to Widgets Bundle Slider and Image Grid type widgets with %link%.', 'siteorigin-panels' ),
+				'url' => SiteOrigin_Panels::premium_url( 'plugin/multiple-media' ),
+				'anchor' => __( 'SiteOrigin Premium', 'siteorigin-panels' ),
```

Comment:
Based on the previous anchor text, this should be `Multiple Media addon`.

Reply:
The text here will read: `Upload multiple image frames at once to Widgets Bundle Slider and Image Grid type widgets with SiteOrigin Premium`. I'm trying to avoid saying multiple twice.